### DISCUSSION TITLE
max loan time

### DISCUSSION THREAD

**Discussion by CarrotCrusadR** [ItzNimaB/sop-it-service-solution#3] [Ideas]
A limit on loan time, half a year would be great. A semester and if need be, it can be prolonged. This will also avoid people doing self service, loaning things in extremely long periods.

**Comment by kenn7575** [comment]
I think this is a good idea and it will certainly be implemented.

**Comment by kenn7575** [comment]
Should this be linked to each product or be a general rule for every type of product?
Pros of loan time property on each product would be that we can set a lower loan time on highly requested and rare products.
